test/specs/scale.category.tests.js

```diff
@@ -465,7 +469,7 @@ describe('Category scale tests', function() {
 
 		var yScale = chart.scales.yScale0;
 		expect(yScale.getPixelForValue({x: 0, y: 2}, 0, 0)).toBeCloseToPixel(257);
-		expect(yScale.getPixelForValue({x: 0, y: 1}, 4, 0)).toBeCloseToPixel(144);
+		expect(yScale.getPixelForValue({x: 0, y: 1}, 4, 0)).toBeCloseToPixel(370);
```

Comment:
Why does this change? Is there a backwards compatibility issue here?

Reply:
The scale was reversed (no reverse defined).
I decided to invert the reverse logic for vertical category scale, to be fully backward compatible.